bin/Powheg/production/2017/5TeV/st_tch_4f_ckm_13TeV/st_tch_4f_ckm_antitop_13TeV.input

```diff
@@ -0,0 +1,74 @@
+! ST-tchannel inputs
+
+#smartsig       1 
+withdamp       1
+hdamp 237.8775 ! 1.379*mtop
+
+withnegweights 0   ! will generate also events with negative-weight 
```

Comment:
have you checked that the negative weighted events are not important, usually we keep the negative weighted event, i.e., use "withnegweights 1" (same comment for top card)
maybe @agrohsje can also comment here since i see he updated 13TeV st card

Reply:
I did not, but I do not see why we should not as you mention. As I said, this comes from UL, but I'll change it in 5 TeV if you prefer.